packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+In enkele gevallen kunnen invoervelden naast elkaar getoond worden, bijvoorbeeld het tonen van een toevoeging naast een huisnummer veld. Dit is echter alleen mogelijk als de velden een directe relatie met elkaar hebben en in een logische volgorde staan.
 
-## Helper text
+##### Voorbeelden van verschillen in breedtes van tekstinvoervelden
 
-Optional helper or feedback text is secondary to the primary action of completing a text entry. This text can be dynamic, such as when a field is filled incorrectly and you want to explain how to correct this error.
+###### IBAN nummer
 
-Helper text is shown below the entry field.
+Een IBAN nummer heeft altijd 18 posities. Dit vaste aantal posities wordt voorgelezen door screenreaders.
 
-## Validation and feedback
+###### Voorbeeld telefoonnummer
 
-A form element should not be validated until the focus of that element is unset. This makes it possible to immediately display an error message if, for example, the entered format does not meet the requirements for that field. As such, a user immediately sees the error and can fix it immediately.
+#### Verzend knop
 
-Feedback should only be shown on incorrect fields in order to minimize the mental strain on users.
+Voorkom dat een eventuele knop om het formulier te wissen geplaatst wordt in nabijheid van de knop om een formulier te versturen. Hierdoor voorkom je het risico dat een bezoeker per ongeluk een forumulier wist.
 
-Not all fields need to be validated; for a first name field we cannot determine whether it is correct. To show this as correctly validated would be strange. When submitting a form, whether something has actually been entered in a field can however be validated. A user does not have to fill in a form according to the visual order on the screen. It should be allowed, for example, to fill in a last name field before filling in the first name field without this being considered an error.
+## Inclusie en toegankelijkheid
 
-## Required vs optional fields
+### Inclusie
 
-If not all fields are required, make clear which fields are.
+#### Gender inclusiviteit
 
-The best way to indicate this is with a text so that it is also announced by screen readers. Are most fields required in a form? Then only show an indication for the optional fields. However, make sure that this pattern is consistent throughout the interface.
+In veel formulieren wordt naar geslacht gevraagd. Soms is dit nodig en is er geen andere optie. Soms wordt dit gedaan om een persoon aan te kunnen spreken. Meestal is de keuze beperkt tot man of vrouw. Probeer echter onnodige vragen over geslacht te vermijden. Waar je absoluut om een geslacht moet vragen, zorg dan tevens voor een derde, neutrale optie. Op advies van o.a. transgender netwerk TNN, kan hiervoor het tekstlabel ‘Niet relevant’ gebruikt worden.
```

Comment:
Ik vraag me af of deze sectie niet gewoon zou kunnen beginnen bij het patroon advies ipv de intro? 

Iets als "Probeer overbodige vragen over geslacht te vermijden. <Voorbeeld van niet overbodige vraag>. Waar je absoluut om een geslacht moet vragen zorg dan naast de beperkte keuze vrouw of man ook voor een neutrale optie. Op advies van o.a. transgender netwerk TNN, kan hiervoor het tekstlabel ‘Niet relevant’ gebruikt worden."

Reply:
Ik denk dat we los hiervan nog een sectie kunnen hebben over het basisprincipe "vraag alleen noodzakelijke persoonlijke gegevens".